<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choose_number</class>
 <widget class="QWidget" name="choose_number">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLineEdit" name="le_choose_number"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string>请输入您想替换的工件号</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pb_choose_sure">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="text">
      <string>请输入1、2或者3</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
